tests/python_functional/functional_tests/destination_drivers/snmp_destination/001-snmp-destination-v2c-normal-working-compared-cisco-ios-traps-default-values.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python
```

Comment:
In pytest by default testcase discovery mechanism collects "test_.*.py" files. So in our case this testcase would not be discovered if I search tests only from top dir ("functions/"). My suggestion for testcase filename: test_snmptrapd_dest_acceptance.py and for testcase name: test_snmptrapd_dest_acceptance
Lets avoid "-" characters

Reply:
file renamed to test_snmp_destination_v2c_cisco_ios_traps.py

Reply:
thanks